libkbfs/config_local.go

```diff
@@ -787,12 +787,23 @@ func (c *ConfigLocal) EnableJournaling(journalRoot string) {
 	log := c.MakeLogger("")
 	listener := c.KBFSOps().(branchChangeListener)
 	jServer = makeJournalServer(c, log, journalRoot, c.BlockCache(),
-		c.BlockServer(), c.MDOps(), listener)
+		c.DirtyBlockCache(), c.BlockServer(), c.MDOps(), listener)
 	ctx := context.Background()
 	err = jServer.EnableExistingJournals(
 		ctx, TLFJournalBackgroundWorkEnabled)
 	if err == nil {
+		syncCache, ok := c.DirtyBlockCache().(*DirtyBlockCacheStandard)
+		if ok {
+			syncCache.name = "sync"
+		}
+		maxSyncBufferSize :=
```

Comment:
Maybe a comment explaining why we set min = max sync buffer size for the journalCache?